SECURITY.md

```diff
@@ -0,0 +1,52 @@
+We would like to ask you to help us protect our users and our systems.
+
+## Please do the following
+
+Submit your findings via the Dutch Government GOV-CERT:
+
+ <https://www.ncsc.nl/contact/kwetsbaarheid-melden>
+
+And reference the "MinVWS/CIO-office, OpenID Connect Package for Laravel" in your report.
+
+Please do not publicly disclose these details without contacting us first. In return we will
+investigate all received vulnerability reports and, where needed, will take appropriate action
+as soon as is warranted.
+
+In most cases this will be a workaround/mitigation first, followed by a fix
+and finally a public disclosure (and CVE number/report at MITRE).
+
+## Do's
+
+- Report the vulnerability as quickly as is reasonably possible, to minimise the risk of hostile actors finding it and taking advantage of it.
+- Report in a manner that safeguards the confidentiality of the report so that others do not gain access to the information.
+- Provide sufficient information to reproduce the problem, so we will be able to resolve it. Usually, the IP address or the URL of the affected system and a description of the vulnerability will be sufficient. 
+But complex vulnerabilities may require further explanation.
+
+## Dont's
+
+- Reveal the vulnerability or problem to others until it is resolved.
+- Build your own backdoor in an information system with the intention of then using it to demonstrate the vulnerability, because doing so can cause additional damage and create unnecessary security risks.
+- Utilise a vulnerability further than necessary to establish its existence.
+- Copy, modify or delete data on the system. An alternative for doing so is making a directory listing of the system.
+- Make changes to the system.
+- Repeatedly gain access to the system or sharing access with others.
+- Use brute force attacks, attacks on physical security, social engineering, distributed denial of service, spam or applications of third parties to gain access to the system.
+
+## What we promise (see <https://www.ncsc.nl/> for full details)
+
+- We strive to respond to your report within 1 business days with our evaluation of the report and an expected resolution date. We will adhere to the NCSC.nl policy of a 60 day disclose and publish.
+- If you have followed the instructions above, we will not take any legal action against you concerning the report.
+- We will not pass on your personal details to third parties without your permission, unless it is necessary to comply with a legal obligation. Reporting under a pseudonym or anonymous is possible.
+- We will keep you informed of the progress towards resolving the problem.
+- In the public information concerning the reported problem, we will give your name as the discoverer of the problem (unless you desire otherwise).
+
+We strive to resolve all problems as quickly as possible, and we would like to play an active role in the ultimate publication on the problem after it is resolved.
```

Comment:
```suggestion
- We will keep you informed of the progress towards resolving the finding.
- In the public information concerning the reported finding, we will give your name as the discoverer of the problem (unless you desire otherwise).

We strive to resolve all findings as quickly as possible, and we would like to play an active role in the ultimate publication on the finding after it is resolved.
```